pycbc/inference/models/relbin.py

```diff
@@ -435,34 +479,42 @@ def _loglr(self):
         p.update(self.static_params)
         wfs = self.get_waveforms(p)
 
-        hh = 0.0
-        hd = 0j
+        norm = 0.0
+        filter = 0j
```

Comment:
'filter' is a reserved name by python, so we try to avoid overloading it new code.